app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -27,7 +27,12 @@
     <h5>
       <b><%= template_invocation.template.name %></b> <%= 'through' %> <%= _(RemoteExecutionProvider.provider_for(template_invocation.template.provider_type)) %>
     </h5>
-    <pre><%= InputTemplateRenderer.new(template_invocation.template, nil, template_invocation).preview %></pre>
+    <% target = template_invocation.targeting.hosts.first %>
+    <% if target.present? %>
+      <%= _('Preview for target %s') % target.name %>
+    <% end %>
+    <pre><%= InputTemplateRenderer.new(template_invocation.template, target, template_invocation).preview %></pre>
```

Comment:
If, for some reason, the template doesn't render the user just sees a "false" - could we show something more informative?